src/main/java/sorald/Constants.java

```diff
@@ -45,4 +68,136 @@
+		sonarChecks.add(DuplicateConditionIfElseIfCheck.class);
+		sonarChecks.add(SynchronizationOnStringOrBoxedCheck.class);
+		sonarChecks.add(HasNextCallingNextCheck.class);
+		sonarChecks.add(IdenticalOperandOnBinaryExpressionCheck.class);
+		sonarChecks.add(LoopExecutingAtMostOnceCheck.class);
+		sonarChecks.add(SelfAssignementCheck.class);
+		sonarChecks.add(StringBufferAndBuilderWithCharCheck.class);
+		sonarChecks.add(MethodNamedHashcodeOrEqualCheck.class);
+		sonarChecks.add(ThreadRunCheck.class);
+		sonarChecks.add(MethodNamedEqualsCheck.class);
+		sonarChecks.add(DoubleBraceInitializationCheck.class);
+		sonarChecks.add(VolatileNonPrimitiveFieldCheck.class);
+		sonarChecks.add(ToArrayCheck.class);
+		sonarChecks.add(AbsOnNegativeCheck.class);
+		sonarChecks.add(IgnoredStreamReturnValueCheck.class);
+		sonarChecks.add(IteratorNextExceptionCheck.class);
+		sonarChecks.add(CompareToResultTestCheck.class);
+		sonarChecks.add(CastArithmeticOperandCheck.class);
+		sonarChecks.add(ShiftOnIntOrLongCheck.class);
+		sonarChecks.add(CompareToReturnValueCheck.class);
+		sonarChecks.add(ImmediateReverseBoxingCheck.class);
+		sonarChecks.add(EqualsArgumentTypeCheck.class);
+		sonarChecks.add(InnerClassOfNonSerializableCheck.class);
+		sonarChecks.add(SerializableSuperConstructorCheck.class);
+		sonarChecks.add(ParameterReassignedToCheck.class);
+		sonarChecks.add(EqualsOverridenWithHashCodeCheck.class);
+		sonarChecks.add(ObjectFinalizeOverridenCallsSuperFinalizeCheck.class);
+		sonarChecks.add(SpringComponentWithWrongScopeCheck.class);
+		sonarChecks.add(ConstructorInjectionCheck.class);
+		sonarChecks.add(ClassWithoutHashCodeInHashStructureCheck.class);
+		sonarChecks.add(InstanceOfAlwaysTrueCheck.class);
+		sonarChecks.add(NullDereferenceInConditionalCheck.class);
+		sonarChecks.add(FloatEqualityCheck.class);
+		sonarChecks.add(IfConditionAlwaysTrueOrFalseCheck.class);
+		sonarChecks.add(ObjectFinalizeCheck.class);
+		sonarChecks.add(GetClassLoaderCheck.class);
+		sonarChecks.add(MathOnFloatCheck.class);
+		sonarChecks.add(SymmetricEqualsCheck.class);
+
+		sonarChecks.add(ObjectOutputStreamCheck.class);
+		sonarChecks.add(NoWayOutLoopCheck.class);
+		sonarChecks.add(UnclosedResourcesCheck.class);
+		sonarChecks.add(DivisionByZeroCheck.class);
+		sonarChecks.add(LocksNotUnlockedCheck.class);
+		sonarChecks.add(StreamConsumedCheck.class);
+		sonarChecks.add(StreamNotConsumedCheck.class);
+		sonarChecks.add(OptionalGetBeforeIsPresentCheck.class);
+		sonarChecks.add(MinMaxRangeCheck.class);
+		sonarChecks.add(ConditionalUnreachableCodeCheck.class);
+		sonarChecks.add(NullDereferenceCheck.class);
+		sonarChecks.add(NonNullSetToNullCheck.class);
+		sonarChecks.add(CustomUnclosedResourcesCheck.class);
+
+//        TEMP_SONAR_CHECKS.add(new DefaultMessageListenerContainerCheck());
+//        TEMP_SONAR_CHECKS.add(new SingleConnectionFactoryCheck());
+//        TEMP_SONAR_CHECKS.add(new DependencyWithSystemScopeCheck());
```

Comment:
Could you please change these commented lines to match the current implementation?

```
//        sonarChecks.add(DefaultMessageListenerContainerCheck.class);
//        sonarChecks.add(SingleConnectionFactoryCheck.class);
//        sonarChecks.add(DependencyWithSystemScopeCheck.class);
```

I don't recall why we have those lines, but I prefer to keep them for now.

Reply:
Right, missed those lines. That would have been confusing later, good spot!